src/MeetingsSDKAdapter.js

```diff
@@ -273,19 +287,26 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @param {string} ID ID of the meeting to mute video
    * @memberof MeetingsSDKAdapter
    */
-  muteVideoMeeting(ID) {
+  async handleLocalVideo(ID) {
     const sdkMeeting = this.fetchMeeting(ID);
+    const state = this.meetings[ID].localVideo.getVideoTracks()[0].enabled;
```

Comment:
Can you name this variable `enabled` or `videoEnabled`? `if/else` is less readable with the word `state`